Dockerfile

```diff
@@ -3,11 +3,10 @@ FROM python:2-alpine
 # Install Docker
 RUN apk add docker curl
 
-# Install Docker Compose
-RUN pip install docker-compose==1.22.0
-
-# Install the AWS CLI
-RUN pip install awscli --upgrade
+# Install Docker Compose, AWS CLI, pipenv
+RUN pip install docker-compose==1.22.0 \
+  pip install awscli --upgrade \ 
```

Comment:
This seems to have caused some problems with the build https://circleci.com/gh/artsy/hokusai/677?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

You can just do `pip install package1 package2` etc

Will follow up with a fix

Reply:
Aha in this case we need `&&`s